PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -138,7 +138,10 @@ class PayPalWebCheckoutClient internal constructor(
      * Call this method at the end of the web checkout flow to clear out all observers and listeners
      */
     fun removeObservers() {
-        activityReference.get()?.let { it.lifecycle.removeObserver(observer) }
+        activity.lifecycle?.let { lifecycle ->
```

Comment:
Sorry I think we want to revert this change - it should be on the activityReference right? And I think Sai is correct it's always non-null, so disregard my comment